### PR TITLE
Add instrument symbol to the 'add trade dialog' header (fix #32)

### LINKED ISSUE #32: Add instrument symbol to the 'add trade dialog' header
E.g. header will now be 'Add Trade (AAPL)'

### PR DESCRIPTION
**Pull request by mike100001**
Third time lucky!

### CODE REVIEW

File: client/src/holdings/AddTradeDialog.tsx [hunk 1]
```diff
@@ -86,7 +86,7 @@ export default function AddTradeDialog({
           }}>
           {({isSubmitting}) => (
             <Form>
-              <DialogTitle id='form-dialog-title'>Add Trade</DialogTitle>
+                <DialogTitle id='form-dialog-title'>Add Trade ({holding.symbol})</DialogTitle>
```
Comment (willbeaufoy) [inline client/src/holdings/AddTradeDialog.tsx]: Nice one, just a small thing, you've added an extra indent where there wasn't one before, so assuming this wasn't intended, can you remove it. Then it's good to go. You can do

```sh
git commit --amend
git push -f origin
```

To amend your existing commit

**Review by willbeaufoy — CHANGES_REQUESTED** [review]

File: client/src/holdings/AddTradeDialog.tsx (hunk 1 above)
Comment (mike100001) [reply client/src/holdings/AddTradeDialog.tsx]: Should be done

Comment (willbeaufoy) [reply client/src/holdings/AddTradeDialog.tsx]: Still looks the same to me, there should be no indentation change from what was there before (unless the autoformatter puts the code on multiple lines). Do you have the 'format on save' on in VS Code? That should fix it. See here: https://github.com/willbeaufoy/portfolio-tracker#code-editor

**Review by willbeaufoy — CHANGES_REQUESTED** [review]

**Comment by mike100001** [comment]
Just did it again twice, I think the first amend one was only amending the message not the commit. Just checked it above and looks good to me.

**Pull request merged**